spec/amp-html-format.md

```diff
@@ -187,13 +173,18 @@ These CSS properties are permanently banned:
 - `behavior`
 - `-moz-binding`
 
-The initial version of AMP will prohibit the use of the following CSS properties. This may be relaxed in the future.
+The following properties are currently blacklisted due to performance concerns:
 
-- `transition`
-- `animation`
 - `filter`
 
-`overflow` (and `overflow-y`, etc.) may not be styled as “auto” or “scroll”. No user defined element in an AMP document may have a scrollbar.
+AMP only allows transitions and animations of properties that can be GPU accelerated in common browsers. We currently whitelist: `opacity`, `transform` (also `-vendorPrefix-transform`).
+
+In the following examples `<property>` needs to be in the whitelist above.
+
+- `transition <property>` (Also -vendorPrefix-transition)
+- `@keyframes name { from: {<property>: value} to {<property: value>} }` (also `@-vendorPrefix-keyframes`)
```

Comment:
Please file a separate issue to whitelist `@keyframes` and `@-vendorPrefix-keyframes`